src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -24,6 +24,7 @@
 #include <mlpack/methods/ann/activation_functions/mish_function.hpp>
 #include <mlpack/methods/ann/activation_functions/lisht_function.hpp>
 #include <mlpack/methods/ann/activation_functions/gelu_function.hpp>
+#include <mlpack/methods/ann/layer/celu.hpp>
```

Comment:
You shouldn't have to do this; it should be included through layer_types.hpp through layer.hpp

Reply:
Hi @birm, thanks for pointing it out. I have done the same.